app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Api
+  class CommentsController < ApplicationController
+    skip_before_action :authenticate_user!, only: %i[index create]
+    skip_before_action :verify_authenticity_token
+    before_action :set_user_post
+
+    def index
+      @comments = @post.comments
+      render json: @comments
+    end
+
+    def create
+      @comment = @post.comments.new(comment_params)
+      @comment.user = @user
+
+      if @comment.save
+        render json: @comment, status: :created
+      else
+        render json: { error: 'Comment not created' }, status: :unprocessable_entity
+      end
+    end
```

Comment:
- You guys have done well with your project so far, but according to the project requirements, when creating a comment, the owner of the comment should be the user that makes it and should not be assigned. This means that your rails API might require authentication, at least to create a comment. I suggest you use the [devise-jwt](https://github.com/waiting-for-dev/devise-jwt) for this. Please fix this 🙏 

![nsrgvklsnrlnglasnrg](https://github.com/George7h/blog/assets/73607512/72a5362f-74d9-4a45-95c0-b06053f45af1)